MyFirstPlugin/META-INF/MANIFEST.MF

```diff
@@ -10,3 +10,5 @@ Require-Bundle: org.eclipse.ui,
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Automatic-Module-Name: MyFirstPlugin
 Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: lib/vavr-0.10.2.jar,
```

Comment:
This is a working approach, but FYI we try to avoid storing binaries in the SCM, so we prefer to use another strategy (pom dependency/p2 dependency).

Reply:
Yes sure, I thought it was just a first step as descrbied in the onboarding page. I will switch to pom dependency.